### ISSUE TITLE
LTC wallet panic during sync

### ISSUE THREAD

**Issue by ukane-philemon** [decred/dcrdex#2616]
I've seen this kind of error several times, for btc and now ltc. 

```
panic: unable to synchronize wallet to chain: unable to perform wallet recovery: couldn't retrieve block b85dcc0bd2c4bdc85b38de07605f130ad1a785b9b063d4ad51362a61ed811615 from network

goroutine 6081 [running]:
github.com/ltcsuite/ltcwallet/wallet.(*Wallet).handleChainNotifications(0xc000220900)
        /Users/philemon/go/pkg/mod/github.com/ltcsuite/ltcwallet@v0.13.1/wallet/chainntfns.go:117 +0xfd3
created by github.com/ltcsuite/ltcwallet/wallet.(*Wallet).SynchronizeRPC in goroutine 180
        /Users/philemon/go/pkg/mod/github.com/ltcsuite/ltcwallet@v0.13.1/wallet/wallet.go:211 +0x249
exit status 2

```

**Comment by buck54321** [comment]
Are you seeing this on btc on master? This should have been fixed by #2396

I just opened a PR for an upstream fix this morning. https://github.com/ltcsuite/ltcwallet/pull/9

**Comment by ukane-philemon** [comment]
I'm seeing this on #2599, I doubt if it has been rebased since the fix was merged.

**Comment by buck54321** [comment]
#2599 was opened way way after #2396 was merged, wasn't it?

**Comment by ukane-philemon** [comment]
I think so, I’m afk until tomorrow but will check.

**Comment by buck54321** [comment]
https://github.com/ltcsuite/ltcwallet/pull/9 is not getting attention, so we'll have to do the same thing with ltcwallet that we did with bchwallet for https://github.com/decred/dcrdex/pull/2670.

**Comment by ukane-philemon** [comment]
> [ltcsuite/ltcwallet#9](https://github.com/ltcsuite/ltcwallet/pull/9) is not getting attention, so we'll have to do the same thing with ltcwallet that we did with bchwallet for #2670.

Okay. 

The dcrlabs repo for ltcwallet looks outdated (205 commits behind upstream).  

Since you already have a PR up, we just need that merged to dcrlabs ltcwallet and use that instead.

**Comment by buck54321** [comment]
Closed by #2743
